content/organization/events/online/index.en.md

```diff
@@ -88,52 +88,52 @@ At the very bottom you should see some buttons -- click on the 'Claim Host' butt
 
 {{< figure src="enter-key.png" alt="Screenshot of the 'enter key' form in the Zoom interface" >}}
 
-### Tips for Running the Zoom meeting
+### Tips for running the Zoom meeting
 
-:eyes: After becoming Host, we suggest that you:
+The guidance in this section comes from the Baltimore, Irvine, NYC, and Santa Barbara chapter guidelines.
```

Comment:
This is a bit tricky here! Some of this content existed before the input of these chapters. Maybe instead we can state something like "These guidance has been updated based on experiences from the Baltimore, Irvine, NYC, and Santa Barbara chapter guidelines, specifically to address issues with trolls in zoom meetings."

Also, if we are going to name the chapters, we should give them the opportunity to review the PR prior to merge.

Reply:
These are good points, and I agree - I've updated my PR to remove this sentence/line and just start with ":eyes: After becoming Host, we suggest that you:..."